src/otx/algo/detection/atss.py

```diff
@@ -52,6 +93,37 @@ def __init__(
         )
         self.image_size = (1, 3, 736, 992)
         self.tile_image_size = self.image_size
+        self._classification_layers: dict[str, dict[str, int]] | None = None
+
+    def _create_model(self) -> nn.Module:
+        from mmengine.runner import load_checkpoint
+
+        config = deepcopy(self.config)
+        self.classification_layers = self.get_classification_layers()
+        model = TorchATSS(**convert_conf_to_mmconfig_dict(config))
```

Comment:
It seems that unlike SSD, torch ATSS model is made in mmdet ATSS model. Am I right?. If then, could you tell me why it is?

Reply:
torchATSS inherits torchSSD. So basically they have same build methods.